### PR TITLE
Avoid periodic cleanup/rectification of changes

### PR DESCRIPTION
**Pull request by kobaska**
### Description

Add a value for which cleanup would not run. Running cleanup on a big database is make the machine run out of memory.

#### Related issues

- None

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: lib/persisted-model.js [hunk 1]
```diff
@@ -1604,7 +1604,9 @@ module.exports = function(registry) {
 
     Model.observe('after delete', rectifyOnDelete);
 
-    if (runtime.isServer) {
+    // Only run if the run time is server
+    // Can switch off cleanup by setting the interval to -1
+    if (runtime.isServer && cleanupInterval !== -1) {
```
Comment (bajtos) [inline lib/persisted-model.js]: Is it perhaps better to use `cleanupInterval > 0`  instead?

**Comment by bajtos** [comment]
@kobaska ping, what's the status of this patch? Are you still keen to get it finished?

**Comment by kobaska** [comment]
@bajtos yes, busy with a release. Will do this and the other pull requests sometime next week.

**Comment by bajtos** [comment]
@kobaska Cool, I totally understand. Good luck with your release! 🍀

**Comment by kobaska** [comment]
@bajtos I have made the changes you requested for

**Comment by bajtos** [comment]
Thank you. I have rebased your patch on top of the latest `2.x`, squashed the commits into a single one and provided a more descriptive commit message (see http://loopback.io/doc/en/contrib/git-commit-messages.html).

Let's wait for CI results before landing this patch.

**Comment by bajtos** [comment]
Landed, thank you for the contribution!

**Pull request merged**